work/worker.go

```diff
@@ -578,6 +581,13 @@ func (self *worker) commitNewWork() {
 			snapshotAccountReadTimer.Update(work.state.SnapshotAccountReads)
 			snapshotStorageReadTimer.Update(work.state.SnapshotStorageReads)
 			snapshotCommitTimer.Update(work.state.SnapshotCommits)
+			if self.chain.Config().IsKoreForkEnabled(header.Number) {
+				newTimer.Update(reward.NewTimer)
```

Comment:
I think It's better to rename the `newTimer` as the thing which we can recognize the hard fork.
If not, we should rename the all the timer names in the next hard fork.

Reply:
@kjeom You're right. Also `oldTimer` seems too generic.
I'll rename `oldTimer` -> `RewardDistributorTimer`, `newTimer` -> `RewardDistributorKip82Timer`.
I'll update `reward.OldTimer` & `reward.NewTimer` as well

Reply:
Thanks for your fast response!